### PR TITLE
[Validate] Fix logging error 

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes:

## Description
fix an issue were error message not log
fix syntax changelog related to this [pr](https://github.com/demisto/demisto-sdk/pull/3323/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 * Fixed an issue where **validate** would not properly detect dependencies of core packs.
 * Removed usages of Random in unit tests to ensure the tests are deterministic.
 * **validate** will now run on all the pack content items when the pack supported marketplaces are modified.
-* Fixed an issue where **pre-commit** running when there was no change to the files
+* **pre-commit** will stop running at the start when there was no change to any files.
```
Comment (dorschw) [inline CHANGELOG.md:12]:
```suggestion
* **pre-commit** no longer runs when there are no modified files (unless provided with input files).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
 * Fixed an issue where errors in **validate** were logged as `info`.
-
+* Fixed an issue where the error messages were not log in **validate**
```
Comment (dorschw) [inline CHANGELOG.md:14]:
```suggestion
* Fixed an issue where **validate** error messages were not logged
```

**Review by tkatzir — APPROVED** [review]
Great catch!

File: CHANGELOG.md (hunk 2 above)
Comment (dorschw) [inline CHANGELOG.md:14]: In what specific cases?

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,10 +9,9 @@
 * Fixed an issue where **validate** would not properly detect dependencies of core packs.
 * Removed usages of Random in unit tests to ensure the tests are deterministic.
 * **validate** will now run on all the pack content items when the pack supported marketplaces are modified.
-* Fixed an issue where **pre-commit** running when there was no change to the files
+* **pre-commit** no longer runs when there are no modified files (unless provided with input files).
 * Fixed an issue where errors in **validate** were logged as `info`.
-
-
+* Fixed an issue where **validate** error messages were not logged when the `param` is not valid or if the default field in reputation command is no valid
```
Comment (dorschw) [inline CHANGELOG.md:14]:
```suggestion
* Fixed an issue where **validate** error messages were not logged when an integration param, or the default argument in reputation commands is not valid.
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**